src/Meshes/explodedMesh.ts

```diff
@@ -0,0 +1,184 @@
+import { Mesh } from "../Meshes/mesh";
+import { Scene } from "../scene";
+import { Vector3 } from "../Maths/math";
+
+/**
+ * Class used to explode meshes.
+ */
+export class ExplodedMesh extends Mesh {

```

Comment:
After giving it a thought I do not think this should inherit from Mesh as this is not really a mesh but more a group of them

Reply:
Thoughts?

Reply:
I agree. Things like position, visibility, etc. are unusable